File: javascript/uri-online-judge/1005.js
const fs = require('fs').readFileSync('/dev/stdin', 'utf-8');
const lines = fs.split(/\n/);

const numberA = Number( lines.shift() );
const numberB = Number( lines.shift() );

const weightA = 3.5
const weightB = 7.5

const ponderedMean = ( numberA * weightA  + numberB * weightB ) / (weightA + weightB);

console.log(`MEDIA = ${ ponderedMean.toFixed( 5 ) }`);

File: java/uri-online-judge/1005.java
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.io.IOException;

public class Main {
  public static void main(String[] args) throws IOException {
    BufferedReader input = new BufferedReader( new InputStreamReader( System.in ) );

    Double numberA = Double.parseDouble( input.readLine() );
    Double numberB = Double.parseDouble( input.readLine() );

    Double weightA = 3.5;
    Double weightB = 7.5;
    Double ponderedMean = (numberA*weightA + numberB*weightB) / (weightA+weightB);

    System.out.printf("MEDIA = %.5f\n", ponderedMean);
  }
}
